### DISCUSSION TITLE
Precision flight competitions

### DISCUSSION THREAD

**Discussion by Ireca737** [SAHorowitz/MSFS2020-PilotPathRecorder#20] [Ideas]
It would be fantastic if it were possible to generate a telemetry report on a database, such as Excel, in order to compare the deviation (speed, altitude, vertical situation and horizontal situation) between the recorded flight report and the planned flight path.
All this to be able to obtain penalty  data deviation  on VFR flights made exclusively with the stopwatch and VFR charts,
It would be a great tool for virtual flight companies to organize flight competitions and ranking the pilot's ability to fly without the aid of an autopilot or radio navigation tools.

**Comment by SAHorowitz** [comment]
Interesting idea. I will need to look again at what can be retrieved for pre-made flight plans. Before msfs2020 could not really share what was in the original plan only what waypoints you crossed that were in a plan.
